packages/renderer/src/lib/deployments/DeploymentDetailsSummary.svelte

```diff
@@ -1,11 +1,10 @@
 <script lang="ts">
 import type { V1Deployment } from '@kubernetes/client-node';
 import ErrorMessage from '../ui/ErrorMessage.svelte';
-import DeploymentColumnConditions from './DeploymentColumnConditions.svelte';
-import type { DeploymentUI } from './DeploymentUI';
-import DeploymentColumnPods from './DeploymentColumnPods.svelte';
+import ObjectMeta from '../kube/details/ObjectMeta.svelte';
```

Comment:
I would prefix ObjectMeta with something like KubernetesDetailsObjectMeta to avoid to have like 5 ObjectMeta classes in Podman Desktop